os/kernel/binary_manager/binary_manager_recovery.c

```diff
@@ -310,11 +325,9 @@ void binary_manager_recovery(int bin_idx)
 	}
 
 #ifdef CONFIG_SUPPORT_COMMON_BINARY
-	/* binid is zero means a crash happened in common library
-	 * We need to reload the library and all the apps
-	 */
+	/* If a fault happened in common library, we need to reload the library and all user binaries */
```

Comment:
Sometimes you used the past(happened), but sometimes present(happens).
Please unify one of them.